lib/commands/migrate.js

```diff
@@ -1,36 +1,37 @@
 "use strict";
 
 const fs = require("fs");
+const path = require("path");
 const chalk = require("chalk");
 const diff = require("diff");
 const inquirer = require("inquirer");
 const PLazy = require("p-lazy");
 const Listr = require("listr");
 
-const validate = require("webpack").validate;
-const WebpackOptionsValidationError = require("webpack")
-	.WebpackOptionsValidationError;
+const { validate } = require("webpack");
+const { WebpackOptionsValidationError } = require("webpack");
 
 const runPrettier = require("../utils/run-prettier");
 
 /**
-*
-* Runs migration on a given configuration using AST's and promises
-* to sequentially transform a configuration file.
-*
-* @param {String} currentConfigPath - Location of the configuration to be migrated
-* @param {String} outputConfigPath - Location to where the configuration should be written
-* @param {Object} options - Any additional options regarding code style of the written configuration
+ *
+ * Runs migration on a given configuration using AST's and promises
+ * to sequentially transform a configuration file.
+ *
+ * @param {Array} args - Migrate options and arguments such as input and
+ * output path
+ * @returns {Promise} Runs the migration using a promise that will throw any errors during each transform
+ * or output if the user decides to abort the migration
+ */
 
-* @returns {Promise} Runs the migration using a promise that will throw any errors during each transform
-* or output if the user decides to abort the migration
-*/
-
-module.exports = function migrate(
-	currentConfigPath,
-	outputConfigPath,
-	options
-) {
+module.exports = function migrate(...args) {
+	const filePaths = args ? args.slice(2).pop() : null;
+	const currentConfigPath = args.slice(2).length === 1 ? [] : [filePaths];
+	if (!filePaths.length) {
```

Comment:
`(!filePaths || filePaths.length)` since filePath can be undefined.
^In case of string, currently it's an array.

Reply:
its a string now, naming is a bit weird

Reply:
Based on line 28, `filePaths` can be `null`. There's need for a second check: `filePaths.length` could throw and error

Reply:
Bingo! @ematipico Good catch